source/find_getmore_killcursors_commands.rst

```diff
@@ -356,6 +347,12 @@ The new **find** command has different semantics to the existing 3.0 and earlier
 
 Once the limit on the cursor has been reached the server will destroy the cursor and return a **cursorId** of **0** in the **OP_REPLY**. This differs from existing **OP_QUERY** behavior where there is no server side concept of limit and where the driver **MUST** keep track of the limit on the client side and **MUST** send a **OP_KILLCURSORS** wire protocol message when it limit is reached.
 
+When using the **find** and **getMore** command the same **batchSize** value **SHOULD** be passed to both commands. This means that if the **find** command had a **batchSize** of `3` the subsequent **getMore** commands **SHOULD** also have **batchSize** set to `3`. 
```

Comment:
Should will stay. Both python and node.js calculate the cursor limit for pre 3.2 and 3.2 and passes that through to the wire protocol implementation. So in the case of limit 4 and batchSize of 3 it will run find with batchSize of 3 and getmore with batchSize of 1. The refactoring required to change this behavior is extensive for no gain. Result is the same.